libraries/ui-library/src/components/six-header/six-header.tsx

```diff
@@ -258,11 +266,18 @@ export class SixHeader {
           'six-header__app-switcher': true,
           'six-header__app-switcher--open': this.isSectionSelected(Section.AppSwitcher),
         }}
+        ref={(el) => (this.appSwitcherContainer = el)}
       >
         <a onClick={this.appNameClicked} class="six-header__selected-app">
           {this.selectedApp}
         </a>
-        <six-dropdown distance={13} skidding={20} placement="bottom-end" ref={this.setupAppSwitcher}>
+        <six-dropdown
```

Comment:
I believe we can achieve the intended functionality by relocating the link to the `six-dropdown`. This should suffice without requiring any modifications other than some CSS adjustments, somewhat like this:

```tsx
        <six-dropdown distance={13} skidding={20} placement="bottom-end" ref={this.setupAppSwitcher}>
          <div slot="trigger">
            <a onClick={this.appNameClicked} class="six-header__selected-app">
              {this.selectedApp}
            </a>
            <six-icon-button name="apps" />
          </div>
          <slot name={Slot.AppSwitcher} />
        </six-dropdown>
```

Reply:
this looks just a little bit easier ;) i'll give it a try!

Reply:
i adapted it and fixed the styles. looks good!